<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Share</class>
 <widget class="QWidget" name="Share">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>460</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>128</height>
      </size>
     </property>
     <property name="text">
      <string>Share</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>This is your hash. You can share it with your friends so they can call you:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLineEdit" name="m_pHash">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="m_pEmail">
     <property name="text">
      <string>Email</string>
     </property>
     <property name="icon">
      <iconset theme="mail-send"/>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="m_pCopy">
     <property name="text">
      <string>Copy</string>
     </property>
     <property name="icon">
      <iconset theme="edit-copy"/>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="m_pPrint">
     <property name="text">
      <string>Print</string>
     </property>
     <property name="icon">
      <iconset theme="document-print"/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>304</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QPushButton" name="m_pConf">
     <property name="text">
      <string>Configure other accounts</string>
     </property>
     <property name="icon">
      <iconset theme="configure"/>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>304</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QPushButton" name="m_pOk">
     <property name="text">
      <string>Start using Ring-KDE</string>
     </property>
     <property name="icon">
      <iconset theme="dialog-ok"/>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>304</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
